### ISSUE TITLE
Authentication : avoid storing the certificate to a file

### ISSUE THREAD

**Issue by zamothh** [CiscoDevNet/intersight-powershell#26]
Hello,

When running the Set-IntersightConfiguration command, we have locally store the API key on the hard drive.

I personally find this not secured, and I have many ways to store that certificate securely : 

- Using the Windows Data Protection API
- Using a third party secret management tool

Instead of pointing to a file, I would like to be able to directly send the certificate from memory.

If we want to use the windows data protection API, here is the code :
```
$Cret | Import-Clixml "E:\Secured\Encrypted.xml"
$Iconfig = @{
	BasePath		= "MyBasePath"
	ApiKeyId		= $Cert.username
	ApiKeyCertificate	= $Cert.getnetworkcredential().password # New variable containing the certificate
	HttpSigningHeader =  @("(request-target)", "Host", "Date", "Digest")
}
Set-IntersightConfiguration @Iconfig
```

Creating the credential can be done that way :
```
$SecureString	= (get-content e:\mycret.txt) -AsSecureString 
$Cert= new-object -typename System.Management.Automation.PSCredential -argumentlist "xxxxxxxxxxxx/yyyyyyyyyy/zzzzzzzzzz", $SecureString 
$Cert| Export-Clixml "E:\Secured\Encrypted.xml"
```

**Comment by vvb** [comment]
@Ghufz Let us look into supporting this.

**Comment by Ghufz** [comment]
We have provided a new parameter called KeyString which accepts apikey string, with the use of this parameter one can pass the apikey in-memeroy. This feature will be available in the relase v 1.0.11-13010  or later

**Comment by Ghufz** [comment]
This enhancement is provided in the release https://www.powershellgallery.com/packages/Intersight.PowerShell/1.0.11.13236
